src/main/java/jenkins/plugins/http_request/auth/CertificateAuthentication.java

```diff
@@ -35,7 +35,10 @@ public CloseableHttpClient authenticate(HttpClientBuilder clientBuilder,
 							credentials.getPassword().getPlainText().toCharArray()).build());
 
 			return clientBuilder.build();
-		} catch (Exception e) {
+		}catch(RuntimeException e){
+			throw new RuntimeException(e);
+		} 
+		catch (Exception e) {
```

Comment:
I don't see the benefit of this change in reducing the likelihood of issues.  Can you explain further or point me to a location where I can read more about the benefit of this change?

Reply:
As spotbug spots this issue of REC_CATCH_EXCEPTION so rather than ignoring that bug in excludes file  i think it's a good approach to make a code change to solve that issue

Reply:
Adding this will catch the runtime exceptions and make the issue solve for reference you can check https://github.com/Alluxio/alluxio/issues/10459

Reply:
@MarkEWaite doesn't it seems good to have this change?